src/utils/screengrabber.cpp

```diff
@@ -259,8 +278,9 @@ QRect ScreenGrabber::desktopGeometry()
 
     for (QScreen* const screen : QGuiApplication::screens()) {
         QRect scrRect = screen->geometry();
-        scrRect.moveTo(scrRect.x() / screen->devicePixelRatio(),
-                       scrRect.y() / screen->devicePixelRatio());
+        // Qt6 fix: Don't divide by devicePixelRatio for multi-monitor setups
```

Comment:
I need to do more testing or thinking about this section and how/if it will impact wayland.